src/components/Homepage/Homepage.js

```diff
@@ -1,21 +1,26 @@
-import { React, useState } from "react";
+import { React } from "react";
+import { Link } from "react-router-dom";
 import Categories from "../Categories/Categories.component";
 import Products from "../Products/Products.component";
 import Slider from "../Slider/Slider";
-import featured from "../../mocks/featured-products.json";
+import { useFeaturedProducts } from "../../utils/hooks/useFeaturedProducts";
 
 function Homepage() {
-  const [featuredProducts, setFeaturedProducts] = useState(featured);
-
+  const { data: featuredProducts, isLoading: featuredProductsIsLoading } =
```

Comment:
I thought about that too, but since I use the `Products` component ion the All Products page too, I wasn't sure If I should add the API calls in the Products component itself since it can also use another data to display.